integration/nevermined/SdkWithoutArtifacts.test.ts

```diff
@@ -0,0 +1,59 @@
+import { assert } from 'chai'
+import { decodeJwt } from 'jose'
+import { config } from '../config'
+import { Nevermined, SearchQuery, Account } from '../../src'
+import { ethers } from 'ethers'
+import { HDNode } from 'ethers/lib/utils'
+
+describe('Sdk working without artifacts', () => {
+  let nevermined: Nevermined
+  let account: Account
+
+  beforeEach(async () => {
+    const configCopy = { ...config }
+    configCopy.artifactsFolder = undefined
```

Comment:
I don't think this is enough. If metamask is not available in the browser `web3ProviderUri` cannot be set, right?